mscore/data/shortcuts.xml

```diff
@@ -491,6 +491,14 @@
     <seq>+</seq>
     </SC>
   <SC>
+    <key>pad-raise</key>
+    <seq>NumPad++</seq>
```

Comment:
Shortcuts are scarse and + is already used. It also makes more sense to have + acting on the duration, like most of the numpad. So I would prefer to have no default shortcut, or other shortcuts.